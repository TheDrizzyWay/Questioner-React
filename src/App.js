import React from 'react';
import { Provider } from 'react-redux';
import store from './store';
import Routes from './routes';
import { ToastContainer } from 'react-toastify';
import 'react-toastify/dist/ReactToastify.min.css';

const App = () => {
    return (
        <Provider store={store}>
            <Routes />
            <ToastContainer position="top-center"
                removeCloseButton autoClose={3000} hideProgressBar />
        </Provider>
    );
};

export default App;
