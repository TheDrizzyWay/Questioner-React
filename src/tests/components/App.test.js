import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import App from '../../App';

describe('App', () => {
    it('Should render component successfully', () => {
        const wrapper = shallow(<App />);
        expect(wrapper.length).toEqual(1);
    });

    test('Should maintain existing snapshot', () => {
        const component = renderer.create(
            <App />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
