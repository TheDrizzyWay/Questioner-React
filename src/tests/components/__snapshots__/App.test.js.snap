// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Should maintain existing snapshot 1`] = `
<div
  className="hpg"
>
  <div
    className="header_container"
  >
    <header>
      <div
        className="header_div"
      >
        <h1>
          Questioner
        </h1>
        <div
          className="details"
        >
          <p>
            Schedule and organize meetups 
            <br />
            with no fuss.
          </p>
          <a
            className="link-style"
            href="/signup"
            onClick={[Function]}
          >
            <button
              name="button"
              type="button"
            >
              Get Started
            </button>
          </a>
        </div>
      </div>
    </header>
  </div>
  <footer>
    <p>
      Powered by TheDrizzyWay Â© 2018
    </p>
    <br />
  </footer>
</div>
`;
